File: Project/Android/User/app/src/main/java/eduku/org/utils/Models/RippleLog.java
package eduku.org.utils.Models;

import java.util.Date;

/**
 * Created by mickey on 2/23/16.
 */
public class RippleLog {
    public String SenderName;
    public String SenderCity;
    public String RecipientName;
    public String RecipientCity;
    public Date SendDate;

    public RippleLog(String _senderName, String _senderCity, String _recipientName, String _recipientCity, Date _date) {
        SenderName = _senderName;
        SenderCity = _senderCity;
        RecipientName = _recipientName;
        RecipientCity = _recipientCity;
        SendDate = _date;
    }
}

File: Project/iOS/User/eduku/Models/RippleLog.swift
//
//  RippleLog.swift
//  eduku
//
//  Created by Mickey on 2/13/16.
//  Copyright © 2016 Mickey. All rights reserved.
//

import Foundation

class RippleLog{
    var SenderName: String?
    var SenderCity: String?
    var RecipientName: String?
    var RecipientCity: String?
    var Date : NSDate?
    
    init(_senderName: String, _senderCity: String, _recipientName: String, _recipientCity: String, _date : NSDate) {
        SenderName = _senderName
        SenderCity = _senderCity
        RecipientName = _recipientName
        RecipientCity = _recipientCity
        Date = _date
        
    }
}